docker/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# See the file "COPYING" in the main distribution directory for copyright.
+
+# Layer to build Zeek. {{{
+FROM debian:buster-slim AS build_zeek
+
+# Configure system for build.
+RUN apt-get -q update \
+ && apt-get install -q -y --no-install-recommends \
+     bind9 \
+     bison \
+     cmake \
+     flex \
+     g++ \
+     gcc \
+     libmaxminddb-dev \
+     libpcap-dev \
+     libssl-dev \
+     libz-dev \
+     make \
+     python3-minimal \
+     python3-dev \
+     swig \
+     ninja-build \
+     python3-pip \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+COPY . /src/zeek
```

Comment:
What's the build context when we run this? The toplevel of the git clone?

Reply:
Yes, exactly. I added a comment here.